### PR TITLE
Migrate pkg/proxy/ipvs/ipset.go logs to structured logging

### PR DESCRIPTION
**Pull request by Nick-Triller**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

xref:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/proxy/ipvs/ipset.go [hunk 1]
```diff
@@ -191,13 +191,13 @@ func ensureIPSet(set *IPSet) error {
 func checkMinVersion(vstring string) bool {
 	version, err := utilversion.ParseGeneric(vstring)
 	if err != nil {
-		klog.Errorf("vstring (%s) is not a valid version string: %v", vstring, err)
+		klog.ErrorS(err, "Invalid version string vstring", "version", vstring)
```
Comment (shubheksha) [inline pkg/proxy/ipvs/ipset.go:194]:
```suggestion
		klog.ErrorS(err, "Invalid version string", "version", vstring)
```

File: pkg/proxy/ipvs/ipset.go [hunk 2]
```diff
@@ -191,13 +191,13 @@ func ensureIPSet(set *IPSet) error {
...
 		return false
 	}
 
 	minVersion, err := utilversion.ParseGeneric(MinIPSetCheckVersion)
 	if err != nil {
-		klog.Errorf("MinCheckVersion (%s) is not a valid version string: %v", MinIPSetCheckVersion, err)
+		klog.ErrorS(err, "Invalid version string MinCheckVersion", "version", MinIPSetCheckVersion)
```
Comment (shubheksha) [inline pkg/proxy/ipvs/ipset.go:200]:
```suggestion
		klog.ErrorS(err, "Invalid version string", "minIPSetCheckVersion", MinIPSetCheckVersion)
```

File: pkg/proxy/ipvs/ipset.go [hunk 3]
```diff
@@ -110,7 +110,7 @@ func NewIPSet(handle utilipset.Interface, name string, setType utilipset.Type, i
 		if strings.HasPrefix(name, "KUBE-") {
 			name = strings.Replace(name, "KUBE-", "KUBE-6-", 1)
 			if len(name) > 31 {
-				klog.Warningf("ipset name truncated; [%s] -> [%s]", name, name[:31])
+				klog.InfoS("Truncated ipset name", "ipsetNameOld", name, "ipsetNameNew", name[:31])
```
Comment (shubheksha) [inline pkg/proxy/ipvs/ipset.go:113]: for consistency with the rest of the file
```suggestion
				klog.InfoS("Truncated ip set name", "ipsetNameOld", name, "ipsetNameNew", name[:31])
```

**Review by shubheksha — CHANGES_REQUESTED** [review]

**Comment by Nick-Triller** [comment]
Hi @shubheksha, thanks for your suggestions. I applied all of them and rebased on master.

**Comment by shivanshu1333** [comment]
Closing as some of the logs are not properly migrated and https://github.com/kubernetes/kubernetes/pull/104932#issuecomment-917808512 got merged which covers migration of this file.
/close

**Pull request closed without merging**